src/options/arith_options.toml

```diff
@@ -551,3 +551,12 @@ header = "options/arith_options.h"
   default    = "true"
   read_only  = true
   help       = "whether to increment the precision for irrational function constraints"
+
+[[option]]
+  name       = "brabTest"
+  category   = "regular"
+  long       = "arith-brab"
+  type       = "bool"
+  default    = "false"
```

Comment:
should we turn it on by default?
@ajreynol @amaleewilson what do you think?

Reply:
Yes, we should enable by default.